app/_how-tos/kic-vault-aws.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Configure an AWS Secrets Manager Vault
+description: "Configure Secrets Manager AWS Vault with {{ site.kic_product_name }} and the KongVault CRD"
+content_type: how_to
+
+permalink: /kubernetes-ingress-controller/vault/aws/
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - /gateway/secrets-management/
+
+min_version:
+  kic: '3.1'
+
+products:
+  - kic
+
+tools:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+entities: []
+
+tldr:
+  q: How do I configure Hashicorp Vault with {{ site.kic_product_name }}?
+  a: Create a `KongVault` CRD and then use the `vault://` reference in your plugin configuration
+
+prereqs:
+  enterprise: true
+  kubernetes:
+    skip_proxy_ip: true
+    gateway_custom_env:
+      AWS_ACCESS_KEY_ID: $AWS_ACCESS_KEY_ID
+      AWS_SECRET_ACCESS_KEY: $AWS_SECRET_ACCESS_KEY
+  inline:
+    - title: AWS configuration
```

Comment:
Make this a shared include prereq file (to share with app/_how-tos/configure-aws-secrets-manager-as-a-vault-backend-with-vault-entity.md)

Reply:
Pushed commits to do this